x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/metric/SecurityMetricType.java

```diff
@@ -38,6 +38,17 @@ public enum SecurityMetricType {
         )
     ),
 
+    AUTHC_OAUTH2_TOKEN(
+        SecurityMetricGroup.AUTHC,
+        new SecurityMetricInfo("es.security.authc.token.success.total", "Number of successful OAuth2 token authentications.", "count"),
```

Comment:
Good question. In both [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html#token-service-settings) and [Kibana](https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#token-authentication) settings it's referred only as a `security.authc.token`. So my intention was to keep the same naming conventions to avoid potential confusions.